Automattic/wp-calypso
# Add superseded state to product cards in pricing page

### Changes proposed in this Pull Request

This PR marks the products that are made irrelevant by a user's subscription as such, in the pricing page. For instance, Backup Daily is of no interest when you own Complete, since Complete includes real-time backups.

Keeping the same example, Backup Daily is not technically included in Complete, but we'll show the existing "Included in plan" message to make it simpler and avoid confusion.

Fixes 1164141197617539-as-1201041449585202

### Implementation notes

I initially tried to check if a plan superseded the other by comparing features. However, this seemed brittle to me in the long run and decided to opt for a config map.

### Testing instructions

_Prerequisite: make sure you have a site with a Complete subscription, and another site with a different subscription_

- Download the PR and run Calypso
- Visit the pricing page with the Complete site
- Notice that Backup Daily, Security Daily, and Security Real-time are marked as included
- Verify that the rest of the page looks the same
- For good measure, select the other site, and check that products cards show the proper status, if any

### Screenshots
<img width="1105" alt="Screen Shot 2021-10-06 at 4 17 39 PM" src="https://user-images.githubusercontent.com/1620183/136279300-c3787c80-616f-435d-aa18-c79b9ea52e9b.png">
